src/Gui/View3DInventorViewer.cpp

```diff
@@ -642,35 +646,43 @@ View3DInventorViewer::~View3DInventorViewer()
     delete glAction;
 }
 
-void View3DInventorViewer::createStandardCursors(double dpr)
+QPixmap View3DInventorViewer::fixupCursor(const QBitmap &bitmap, const QBitmap &mask, double dpr) const
```

Comment:
As this has no reference to the `this` object it probably could be a private function of this file and so it would not need to be defined in the `.h` file. I'm living this only as a sidenote however, this way is also fine by me.

Reply:
Indeed. I did it this way for consistency as I didn't find any other private function in that file, but I was obviously not looking hard enough. After test build finishes here, I'll push this change as well.